enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/RequestContextConverter.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2023 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.impl.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+
+/**
+ * Converter for HTTP request context information.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+// TODO (racansky, 2023-01-09) duplicates com.wultra.app.enrollmentserver.impl.service.converter.RequestContextConverter, extract to lime-java-core
```

Comment:
Enrollment server without onboarding features does not depend on `enrollment-server-onboarding-common`, so we do not have any other reasonable _common_ module. `lime-java-core` has been already released. Let's keep duplicity here for a while and extract it later on.

Reply:
OK.

Reply:
See https://github.com/wultra/lime-java-core/issues/151